cob_bringup/drivers/sick_visionary_t.launch

```diff
@@ -13,6 +13,6 @@
 		<remap from="$(arg name)_node/intensity" to="$(arg name)/depth/intensity"/>
 		<remap from="$(arg name)_node/ios" to="$(arg name)/depth/ios"/>
 		<remap from="$(arg name)_node/points" to="$(arg name)/depth/points"/>
-	</node>
+	</node-->
```

Comment:
I do not feel responsible for this repo anymore...all robots using it, could stay on the `indigo_dev` branch and still use `kinetic`